index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="app-wrap">
+    <header class="header1">
+        <img src="Images/My Logo.png">
+        <img src="Images/menu.png">
+    </header>
+<header class="content">
+    <div class="header2">
+        <h1>I'm Mounir Glad to see you ! <br>I'm a software developer </h1>
+        
+        <p> I can help you build a product , 
+            feature or website Look through 
+            some of my work and experience! 
+            If you like what you see and have 
+            a project you need coded, 
+            
+            don’t hestiate to contact me</p>
+        <h3>LET'S CONNECT</h3>
+    </div>
+
+    <div class="icon-bar">
+        <a href="#">
+            <img src="Images/twitter.png" >
+        </a>
+        <a href="#">
+         <img src="Images/Linkedin icon.png" >
+        </a>
+        <a href="#">
+            <img src="Images/medium.png" >
+        </a>
+        <a href="#">
+        <img src="Images/github.png" >
```

Comment:
- [ ] Kindly stick with the Figma design as much as possible (color, font family, size, and so on).  your design is totally different from Figma
### Your design
![yours ](https://user-images.githubusercontent.com/81727988/164934197-37c60bd4-47c2-43ae-a347-c4a8388e20e2.png)

### Figma design
![Figma-design](https://user-images.githubusercontent.com/81727988/164934120-35ce72c9-0f3b-482e-b50d-3fd74e1bd311.png)